ti-auth

```diff
@@ -0,0 +1 @@
+authenticate with ti before install the github collector
```

Comment:
screenShot added for reference
![newDesign](https://user-images.githubusercontent.com/100127824/154945277-f2a2450a-49fc-4535-b3ca-8f4f6bb0d40a.PNG)